v2/samples/azureiotedge-simulated-temperature-sensor/Docker/linux-arm/Dockerfile.debug

```diff
@@ -0,0 +1,15 @@
+FROM microsoft/dotnet:2.0.0-runtime-stretch-arm32v7
+
+ARG EXE_DIR=.
+
+WORKDIR /app
+
+COPY $EXE_DIR/ ./
+
+RUN apt-get update
+
+RUN apt-get install -y unzip procps
+
+RUN curl -sSL https://aka.ms/getvsdbgsh | bash /dev/stdin -v latest -l ~/vsdbg
```

Comment:
Does this work on ARM platforms?

Reply:
no, I removed the `.debug` version in the arm folder